src/rgw/rgw_rest_client.cc

```diff
@@ -61,34 +61,35 @@ int RGWHTTPSimpleRequest::receive_header(void *ptr, size_t len)
       *p = '\0';
       ldpp_dout(this, 30) << "received header:" << line << dendl;
       // TODO: fill whatever data required here
-      char *l = line;
+      char *l = line.data();
```

Comment:
Can we also spell char* as auto l = begin(line)? (It would be pretty, but I'd rather get VLAs than have to see the whole loop "modernized" as that can get into a time sink.)

Reply:
Rather not when I'm mixing in uses of strsep and other things that explicitly spell it `char*`.